src/opera/pge/cslc_s1/cslc_s1_pge.py

```diff
@@ -171,14 +200,24 @@ def _cslc_filename(self, inter_filename):
         """
         core_filename = self._core_filename(inter_filename)
 
-        cslc_metadata = self._collect_cslc_product_metadata()
+        burst_id = Path(os.path.dirname(inter_filename)).parts[-2]
+        burst_id = burst_id.upper().replace('_', '-')
```

Comment:
I'm wondering why the replace is necessary.  Are we having to do this because something is being done incorrectly upstream?  If so, it seems we should comment on why the replace is being done.

Reply:
This is done to prepare the burst ID for inclusion in the output filename. The current convention is for the burst ID to be all caps and use hyphens for separators. I agree that this should be documented, will update.